tests/utilities/streaming_data_measurment/StreamingMeasurement.serviceconfig

```diff
@@ -0,0 +1,18 @@
+{
+  "services": [
+    {
+      "displayName": "Sample Streaming Measurement (Py)",
+      "serviceClass": "ni.examples.SampleStreamingMeasurement_Python",
```

Comment:
Update the display names and service classes to be consistent with the new names. Replace "examples" with "tests".

Reply:
Updated display names and service class with new names.